app/controllers/HomepageController.scala

```diff
@@ -36,7 +36,7 @@ class HomepageController @Inject()(val controllerComponents: SecurityComponents,
       } else if(request.token.isStandardUser){
         Ok(views.html.standard.homepage(request.token.name))
       } else {
-        Ok(views.html.registrationComplete(request.token.name))
+        Ok(views.html.registrationComplete(request.token.name, request.token.isJudgmentUser))
```

Comment:
Isn't this page only showed to judgment users? And won't `request.token.isJudgmentUser` always be false as well because where it's true is covered in the first `if` So we'd have a judgments only page with `judgmentsUser = false` passed to it.
I might be talking nonsense though.

Reply:
It's actually shown to any user without a user type. Given that at the moment this only applies to the judgment users, have removed this, but changed the html template to only show the judgment's faqs. Which is what I think Kirsten wants.